input/pagecontent/reference.md

```diff
@@ -680,9 +686,9 @@ The following keywords (case-sensitive) are defined:
 |----------|---------|---------|
 | Description | Provides a human-readable description | string or markdown |
 | Expression | Provides a FHIR path expression in an invariant | FHIRPath string |
-| Id | Provides a identifier for an item | id |
+| Id | Provides an identifier for an item | id |
 | InstanceOf | Names the profile or resource an instance instantiates | name or id or url |
-| Parent | Names the base class for a profile or extension | name or id or url |
+| Parent | Names the base definition for a profile or extension | name or id or url |
```

Comment:
The term `class` isn't really a formal term in FHIR.  In the SD, this field is actually called `baseDefinition`, so I think "base definition" makes sense and still gets the point across to the casual reader.